project/api/models/activity.py

```diff
@@ -36,3 +36,11 @@ class Activity(models.Model):
 
     created = models.DateTimeField(auto_now_add=True, blank=True, null=True)
     last_modified = models.DateTimeField(auto_now=True, blank=True, null=True)
+
+    @property
+    def is_voted_positively(self):
```

Comment:
For an API, `is_positive_vote` and `is_negative_vote` sound a bit more 'natural'.